### ISSUE TITLE
Custom rest point is not called at all

### ISSUE THREAD

**Issue by rcshubhadeep** [doccano/doccano#2085]
How to reproduce the behaviour

I am not so sure but I will describe in details what is happening here. 

I have two EC2 machines (Both Ubuntu). 

In one I am running doccano (I have started it using the vanilla docker command - `docker pull doccano/doccano` which probably means I have pulled the latest)

Then I uploaded some data. And did a bit of annotation. 

After that I have trained a model based on that annotated data and then hosted the model in the second EC2 instance and created a very simple API using FastAPI on top of it. 

That api is accessible from my local.

That API is accessible **when I am setting up auto labelling** 
![Screenshot from 2022-12-06 22-19-50](https://user-images.githubusercontent.com/5349112/205972790-f5c38632-54db-4ae7-82d3-16244d4f2eb2.png)

And the rest of the steps to set up auto labelling works perfectly fine

mapping step
![Screenshot from 2022-12-06 22-21-51](https://user-images.githubusercontent.com/5349112/205973184-0486ac71-3aee-4020-8484-4a6a7d6a8b17.png)

final step

![Screenshot from 2022-12-06 22-23-13](https://user-images.githubusercontent.com/5349112/205973539-96f643d9-aefc-4e8d-aa0e-61dc6e4226e8.png)

After all this it looks like this - 

![Screenshot from 2022-12-06 22-24-47](https://user-images.githubusercontent.com/5349112/205973895-7e6ff8bd-5a50-42a8-8a96-488d6500d895.png)

As far as I understood from the official tutorial and the thread [here](https://github.com/doccano/doccano/issues/1417) this should be all I need to do.

However once all these are done, the most strange thing starts to happen. I go to the datasets and open one document. And the click on the magic wand. It shows that it is switched off. I slide it on. And ...

NOTHING HAPPENS!! 

I checked the model server log and as well and doccano server log. NO  ACTIVITY. 

So it is like this - 

![Screenshot from 2022-12-06 22-30-02](https://user-images.githubusercontent.com/5349112/205975188-3c4aa9c0-c583-447e-bab1-005b3eb5db6c.png)

And no labels show.

I switch it on

And no labels show.

No log in server both doccano and the model one.

HOWEVER,

When I was setting this up and I tested the result I did see the log in the server, meaning doccano could actually communicate successfully. 

I have checked the chrome developer window. I see nothing (I am using Brave).

What is going wrong here? What am I missing?

---------

Your Environment
---------

*   Operating System: Ubuntu 20.04
*   Python Version Used: I am not sure whatever comes with the default docker image
*   When you install doccano: I installed it 8 weeks ago
*   How did you install doccano (Heroku button etc): Using docker.

**Comment by druskacik** [comment]
The website is not behaving intuitively from the UX perspective. When you turn on Auto Labeling, just try going to the next example (arrow on the top right of the page) and your Auto Labeling API should be called. Then go back and it will be called on the first example. Also, it's called only on examples that are not marked as "labeled".

**Comment by rcshubhadeep** [comment]
You are right. It does not behave at all intuitively. I think this can be easily fixed, No? I mean unless you know this it can be very strange for new users.
